core/util/src/mill/util/Jvm.scala

```diff
@@ -495,7 +495,7 @@ object Jvm {
       }
       .pipe { cache =>
         if (ctx.fold(false)(_.offline)) cache.withCachePolicies(Seq(CachePolicy.LocalOnly))
-        else cache
+        else cache.withCachePolicies(Seq(CachePolicy.FetchMissing))
```

Comment:
CI seems to be green [here](https://github.com/alexarchambault/mill/actions/runs/14642052919/job/41087262442) (mostly, the red is transient GraalVM issues on Windows that should be fixed by https://github.com/com-lihaoyi/mill/pull/4917).

Once https://github.com/coursier/coursier/pull/3389 is merged and released, I'm planning to remove this line, and this PR should be good to review and merge.

Reply:
That's going to need https://github.com/coursier/coursier/pull/3392 too